packages/api/models/granules.js

```diff
@@ -277,6 +300,35 @@ class Granule extends Manager {
 
     return Promise.all(done);
   }
+
+  /**
+   * return the queue of the granules for a given collection,
+   * the items are ordered by granuleId
```

Comment:
Same concern as above, but how can we be sure that they're ordered be granuleId ?